keps/sig-network/2091-cluster-network-policy/README.md

```diff
@@ -232,13 +249,13 @@ relative priority ordering of all configured rules and CNP instances.
 
 #### Rule identifiers
 
-In order to collect traffic statistics corresponding to
-a rule, it is necessary to identify the rule which allows/denies that traffic.
+In order to collect traffic statistics corresponding to a rule in the future,
```

Comment:
I still think we should remove this mention of "traffic statistics" and instead just argue that it's easier for readability's sake + future proofing